CMakeLists.txt

```diff
@@ -438,6 +438,9 @@ if(${CMAKE_CXX_COMPILER_ID} MATCHES "GNU" OR
             NOT (${CMAKE_CXX_COMPILER_VERSION} VERSION_LESS "5.0.0"))
         add_compile_options(-Wno-unused-local-typedefs)
     endif()
+    # There are so many warnings about using GCC > 7.1. It should be mostly
+    # harmless, so we can disable it. 
+    add_compile_options(-Wno-psabi)
```

Comment:
Driveby comment while you develop, but isn't this *quite* hamful? Disabling ABI checks on a binary-distributed library codebase like OpenSim for all builders is a no no.

If you're doing something like cross-compiling with a weird compiler etc. etc. and you want this flag for your build specifically then you can specialize in your buildscript by setting the `CXXFLAGS` environment variable:

https://www.google.com/search?client=firefox-b-d&q=CXXFLAGS

Reply:
Oh, that is a much better option. I'll revert that commit.